requirements.txt

```diff
@@ -1,35 +1,48 @@
+appdirs==1.4.3
 appnope==0.1.0
 backports.shutil-get-terminal-size==1.0.0
 coverage==4.1
 coveralls==1.1
 decorator==4.0.10
-dj-database-url==0.4.1
-Django==1.10
+dj-database-url==0.4.2
+dj-static==0.0.6
+Django==1.10.7
```

Comment:
Please, for learning sake, let's know the issues the former version couldn't handle.